test/Sentry.Diagnostics.DiagnosticSource.Tests/Sentry.Diagnostics.DiagnosticSource.Tests.csproj

```
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0;netcoreapp3.1;netcoreapp2.1;net461</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1' or $(TargetFramework.StartsWith('netstandard2')) or $(TargetFramework.StartsWith('net4')) or $(TargetFramework.StartsWith('net4'))">
```

checking net4 twice: 

```suggestion
  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1' or $(TargetFramework.StartsWith('netstandard2')) or $(TargetFramework.StartsWith('net4'))">
```